practices/coordination.md

```diff
@@ -12,48 +12,94 @@ Here are the goals we optimize for in organizing our team coordination practices
+- {guilabel}`Blocked/Waiting`: Deliverables for which we are waiting for some action that is out of our team's immediate control.
+- {guilabel}`Done`: Deliverables that have been completed. We should close these issues and celebrate the improvements that we have made!
 
 :::{note}
-Deliverables are generally written as **user stories** - that is, with a particular user in mind, and a particular outcome they want to see.
-For example, "As a 2i2c Hub User, I want to install a package on my own."
-Deliverables should be accomplishable in one or two sprints, and may have many tasks associated with them.
+In general, there should be only two or three deliverables per team member, per column on this board.
+It should not become so heavily-populated that it is hard to keep track of deliverables!
 :::
 
-## Tasks are steps to complete a deliverable
+There are two Project Backlogs at 2i2c:
 
-In order to meet a deliverable, we need to take a set of actions, called **Tasks**.
-Tasks are verbs, not outcomes, and they should be extremely concrete and obvious what needs to be done.
-In addition, they should be accomplishable in one setting.
+- The [2i2c Development Projects Backlog](https://github.com/orgs/2i2c-org/projects/7) contains deliverables for our development-focused projects.
+- The [Organizatipnal Project Backlog](https://github.com/2i2c-org/meta/projects/6?fullscreen=true) contains deliverables that are organization-wide and not related to development.
 
-Tasks are the things that 2i2c team members *plan to do* in a sprint.
-This helps keep our actions concrete, while still working towards a deliverable.
+(coordination:deliverables)=
+### Deliverables
 
-### Keeping track of our goals
+Deliverables represent incremental amounts of value we can deliver to a particular stakeholder, and should be completable in a week or two. 
 
-:::{warning}
-We are considering phasing out team goals, and simply using **Projects** and **Deliverables** as described above.
-:::
+Most issues in our repositories are deliverables, in varying states of readiness. When a deliverable is first created, it may lack information, be improperly scoped, or have an unclear path to implementation. We improve this through _issue refinement_ (see below).
 
-Team goals are high-level goals (AKA, there's no single action that will "complete" them) and span a longer period of time.
-Goals generally contain pointers to other issues / PRs / discussions where the work of more specific tasks is getting done.
-You should regularly communicate with other team members about which goals you're focusing on (e.g., via Slack, GitHub Issues, or via the [](coordination:team-syncs))
+A deliverable is ready to work (and can thus be added to the Project Backlog) when it has the following properties (adapted from the [INVEST methodology](https://agileforall.com/new-to-agile-invest-in-good-user-stories/)).
```

Comment:
Ah good point. My one concern is that if we automatically add anything w/ `Deliverable` on it, then it'll become extremely long (depending on how many deliverables we have across all of our repositories). What do you think about this?

Reply:
Hmmm I'm a bit confused - are you commenting regarding automation associated with labels or the suggested change about "added" vs "advanced to that column"?

If we don't make this update, I'm considering how the column in the backlog about discussion is meant to be used. First add ready for work, and then it can transition back to needs discussion/refinement?

Reply:
@choldgraf regarding automation of labels, I'm not sure. If we don't, it means we don't really keep track of our deliverables in one place as i imagined the project backlog would do.

That would be fine, but it raises a question about what deliverables and when to add/remove them from the project board.

Reply:
I think I had imagined something like:

|               | Need more info                     | Ready for Work       |
|---------------|------------------------------------|----------------------|
| Low Priority  | Stays in issues                    | Put in Ready to work |
| High Priority | Put in Needs Discussion/Refinement | Put in Ready to work |

However, I think you are suggesting a simpler model, like:

|               | Need more info                     | Ready for Work       |
|---------------|------------------------------------|----------------------|
| Low Priority  | Put in Needs Discussion/Refinement | Put in Ready to work |
| High Priority | Put in Needs Discussion/Refinement | Put in Ready to work |

Is that right? If so, I think I'm 👍 on trying your approach out, and seeing how it goes. We can use GitHub labels to tag priority high/low. What do you think?

Reply:
Beautiful overview! Yes, that is what I meant and I now have a clear grasp on what I didn't before about what you meant!

I think both options make sense as long as they were a bit explicit, like both options now are.

I've just assumed it would be plausible to setup such automation, but I haven't done it. I think we should go for the non-automated approach until its proven viable. Looking into things I found some indication of viability but not by GitHub's official setup, but through https://github.com/philschatz/project-bot.

Reply:
Quick point on automation: I think the automation would be quite easy using ghapi (https://ghapi.fast.ai/) for example.